src/generate/register.rs

```diff
@@ -277,7 +277,7 @@ pub fn fields(
             let pc_r = Ident::from(&*format!("{}R", pc));
             let _pc_r = Ident::from(&*format!("{}_R", pc));
             let pc_w = Ident::from(&*format!("{}W", pc));
-            let _pc_w = Ident::from(&*format!("_{}W", pc));
+            let _pc_w = Ident::from(&*format!("{}_W", pc));
```

Comment:
Ugh... It's a pity we didn't notice this earlier. But would you please remove it from this PR?

Reply:
Ok.
Then
`_pc_r  => "_{}R"` ?

Reply:
No, just remove the change from this PR and put it in a separate one.